modules/ROOT/examples/live-demos/ui-mode/index.js

```diff
@@ -0,0 +1,51 @@
+tinymce.init({
+    selector: 'textarea#iframe',
+    plugins: 'quickbars table image link lists media autoresize help',
+    toolbar: 'undo redo | blocks | bold italic | alignleft aligncentre alignright alignjustify | indent outdent | bullist numlist',
+    content_style: 'body { font-family:Helvetica,Arial,sans-serif; font-size:16px }',
+    ui_mode: 'split',
```

Comment:
```suggestion
    selector: 'textarea#iframe',
    plugins: 'lists help',
    toolbar: 'undo redo | blocks | bold italic | bullist numlist',
    ui_mode: 'split',
```
Might be better to simplify these demos to just basic editors with only the new option, since this html and js content will be visible in the page